README.md

```diff
@@ -220,106 +216,15 @@ autoscaling policies.
 * **`min_capacity`** - (Required) Minimum task count for autoscaling (this will also be used to define the initial
   desired count of the ECS Fargate Service)
 * **`max_capacity`** - (Required) Maximum task count for autoscaling
+* **`cpu_percentage_target`** (Optional) What the desired average CPU percentage is
```

Comment:
and don't you need the scale_in_cooldown and scale_out_cooldown here?

Reply:
Yes

Reply:
Not sure still if the variables are named well and if they should be in their own block

Reply:
Added